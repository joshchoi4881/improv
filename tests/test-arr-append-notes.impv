func int main() {
  note[] n1;
  note[] n2;
  note[] n3;

  n1 = [<1, "wh">, <5, "wh">];
  n2 = [<4, "wh">];

  n1 = append(n1, n2);
  printNoteArr(n1);

  n1 = append(n1, [<2, "hf">]);
  printNoteArr(n1);

  return 0;
}
