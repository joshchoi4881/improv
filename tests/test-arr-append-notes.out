[<1, wh>, <5, wh>, <4, wh>]
[<1, wh>, <5, wh>, <4, wh>, <2, hf>]
